### ISSUE TITLE
Installing and Administering PECE: Dead link in Quick Install

### ISSUE THREAD

**Issue by ibara** [PECE-project/drupal-pece#43]
The troubleshooting link is dead:
https://pece.readthedocs.io/en/latest/installation.html#troubleshooting

To be un-dead-ified!

**Comment by revagomes** [comment]
https://pece-project.github.io/drupal-pece/installation/#other-post-installation-troubleshooting

**Comment by MegaKeegMan** [comment]
Are the read the docs just down because they are hosted here instead?
https://pece-project.github.io/drupal-pece/installation/

**Comment by mikefortun** [comment]
Yes, that's where they are posted -- not sure where you are saying they are
down?

**Comment by MegaKeegMan** [comment]
The readthedocs link in the original comment is what is referenced in the readme, and is dead.

**Comment by mikefortun** [comment]
oh ok, i thought you were referring to Renato's link in the first message
in this chain, but now I went to GitHub and see what you mean -- yes, we
moved everything over to that new link a while ago now, so you are looking
at the most recent version...
